src/themes/light-theme-with-move.js

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import createTheme from './creator';
+import {primitives as lightThemePrimitives} from './light-theme-primitives';
+
+const primaryFontFamily =
+  'UberMoveText, "Open Sans", "Helvetica Neue", Helvetica, sans-serif';
+
+export const LightThemeMove = createTheme(
+  {
+    ...lightThemePrimitives,
+    primaryFontFamily,
+  },
+  {
+    typography: {
+      font200: {
+        fontFamily: primaryFontFamily,
+        fontSize: '12px',
+        fontWeight: 'normal',
+        lineHeight: '20px',
+      },
+      font250: {
+        fontFamily: primaryFontFamily,
+        fontSize: '12px',
+        fontWeight: 'bold',
+        lineHeight: '20px',
```

Comment:
these are the same as in the default theme, no need to override

Reply:
yeah, I agree - the reason I still added them here is for clarity, to make sure who wants to quickly understand what fonts the UberMoveTheme is using will know instantly

does it make sense?

Reply:
yeah, makes sense